mezzanine/utils/conf.py

```diff
@@ -19,9 +20,17 @@ class SitesAllowedHosts(object):
     def __iter__(self):
         if getattr(self, "_hosts", None) is None:
             from django.contrib.sites.models import Site
-            self._hosts = [s.domain.split(":")[0] for s in Site.objects.all()]
+            try:
+                self._hosts = [
+                    s.domain.split(":")[0] for s in Site.objects.all()
+                ]
+            except OperationalError:
+                return iter([])
         return iter(self._hosts)
 
+    def __add__(self, other):
+        return list(self) + other
+
```

Comment:
Django 1.10 appends ['testserver'] to `ALLOWED_HOSTS` while running tests, and does so before migrations have been run. So SitesAllowedHosts needs to handle addition with a list, and its database table being missing.